File: StatusBarSpeedometer/app/src/main/java/ch/rmy/android/statusbar_tacho/units/MetersPerSecondUnit.java
package ch.rmy.android.statusbar_tacho.units;

import ch.rmy.android.statusbar_tacho.R;

public class MetersPerSecondUnit extends Unit {

    @Override
    public int getNameRes() {
        return R.string.unit_ms;
    }

    @Override
    public float convertSpeed(float metersPerSecond) {
        return metersPerSecond;
    }

}

File: StatusBarSpeedometer/app/src/main/java/ch/rmy/android/statusbar_tacho/units/MetersPerSecondUnit.kt
package ch.rmy.android.statusbar_tacho.units

import ch.rmy.android.statusbar_tacho.R

class MetersPerSecondUnit : Unit() {

    override val nameRes: Int
        get() = R.string.unit_ms

    override val maxValue: Int
        get() = 50

    override val steps: Int
        get() = 5

    override fun convertSpeed(metersPerSecond: Float): Float {
        return metersPerSecond
    }

}
